xla/service/gpu/gpu_device_info_test.cc

```diff
@@ -82,9 +90,61 @@ TEST(DeviceInfoTest, DeviceInfoIsCorrect) {
                              /*l2_cache_size=*/4 * 1024 * 1024,
                              /*clock_rate_ghz=*/::testing::Ge(1.4),
                              /*device_memory_size=*/17'066'622'976));
-  } else {
-    VLOG(1) << "Not tested for " << name;
+  } 
+#if TF_ROCM_VERSION >= 50500
```

Comment:
As I write this now I'm not so sure. Let me know. I'm only considering this because our linter takes issue with how the braces end up with this change, and I'd prefer to not guard this with a `#if ... #endif` if possible. If it is necessary, I'll patch in the necessary `// NOLINT` lines internally

Reply:
Hi @ddunl Thanks a lot for the feedback.

Yes, this ```#if``` is required here. As prior to ROCm 5.5, the device_name cannot be printed out and we have to wait ROCm5.5 is released to fix this test.

Reply:
Gotcha, I may not get the internal patch done by the end of the day, if not it'll be early tomorrow. Thanks for letting me know!!

Reply:
I have adjusted the patch internally to have a // NOLINT at the end of these lines. There was also a missing std:: in front of string. With these fixes, it should be ready to land, I will just need to wait for an approval for the adjusted patch.

Reply:
@akuegel Thanks a lot!